knowledge_base/002_TNS_Could_Not_Resolve_Connect_Identifier_Error_Fix.md

```diff
@@ -0,0 +1,22 @@
+# "ORA-12154: TNS:could not resolve the connect identifier specified" Error
+
+## This error message commonly shows up for the following 2 reasons:
+1. Incorrect wallet configuration.
+   1. Follow [instructions here](https://github.com/oracle/oci-data-science-ai-samples/blob/master/environment_examples/database/autonomous_database.ipynb), if you haven't already, to download the wallet and unzip it locally.
+   2. Next, within `~/.database` directory there should be a folder with your database name. Within that will be the unzipped wallet files with some modifications made by `ads` code.  Here are two things to confirm are correct within the wallet files:
```

Comment:
"Within that will be the unzipped wallet files with some modifications made by `ads` code" Change to "Within your database name directory, your database configuration files."

Reply:
Updated!